test.sh

```diff
@@ -145,7 +144,6 @@ if [[ "$RUN" =~ "generate" ]] ; then
   go install ./probs
   go install ./vendor/google.golang.org/grpc/codes
   run_and_expect_silence go generate ./...
-  run_and_expect_silence git diff --exit-code $(ls | grep -v Godeps)
```

Comment:
Deleting the line here is the wrong thing; the git diff is actually the core of what this is testing. Instead, just replace the `$()` with `.`.